functions-iterators/starter.js

```diff
@@ -29,42 +27,168 @@ This will help us track your progress and it will give you practice writing good
 
 // PART 0: Just DO it!!
 
+function justDoIt (name){
+    console.log(name, 'just do it!')
+}
+justDoIt('AK');
 
 
 // PART 1: Big or Small String?
 
+function bigOrSmallString(word){
+    if (word.length > 10){
+        console.log('This word is loooooong!');
+    } else {
+        console.log('This word is short');
+    }
 
+}
 
 // PART 2: Odd or Even String Length?
 
+function oddOrEvenString (word) {
+    if (word.length %2 == 0)
+          {console.log('It is even number');
+          }
+   
+      else {
+          (word.length %2 != 1)
```

Comment:
if you want to use condition with an else statement you use `else if ( condition )`